Magento_Theme/templates/footer/footer-links.phtml

```diff
@@ -24,7 +25,7 @@
                     <use xlink:href="<?= $block->getViewFileUrl('/images/icons-sprite.svg#angle-down') ?>"></use>
```

Comment:
please add `role="presentation"` or `aria-hidden="true"` to all icons in dropdown-list, IMO there is no need to show them to screen reader users. If we hide them, there is no need to add the `<title></title>` or `role="img"`

Reply:
`role="presentation"` added to each icon inside dropdown-list, `<title>` removed in such cases